stac_fastapi/extensions/stac_fastapi/extensions/core/transaction.py

```diff
@@ -132,6 +135,11 @@ def register_delete_item(self):
             endpoint=create_async_endpoint(self.client.delete_item, ItemUri),
         )
 
+    def register_patch_item(self):
+        """Register patch item endpoint (PATCH
+        /collections/{collection_id}/items/{item_id})."""
+        raise NotImplementedError
```

Comment:
Make it a bit more clear that the `Patch` method are not implemented by purpose!